pkg/util/mon/bytes_usage.go

```diff
@@ -686,9 +684,72 @@ type BoundAccount struct {
 	mon      *BytesMonitor
 
 	earmark int64
+}
+
+// ConcurrentBoundAccount is a thread safe wrapper around BoundAccount.
+type ConcurrentBoundAccount struct {
+	BoundAccount
```

Comment:
so, I started with:
```
type ConcurrentBoundAccount struct {
  syncutil.Mutex
  wrapped BoundAccount
}
```
BUT, this way I need to implement all the methods for `BoundAccount` (I think it's `Init/Empty/Clear/Close`, not too bad), instead of just those that I need to change (those that need a lock). IOW I want to inherit from `BoundAccount`.
that said I ok with either way..

Reply:
You don't want to hold the lock when calling empty/clear/close?

Reply:
mmm, good point.. those don't use the lock today but they actually should! I'll fix.

Reply:
Done, PTAL.